packages/app/codegen/subgraph.ts

```diff
@@ -1,11 +1,17 @@
-import { gql } from 'urql';
-import * as Urql from 'urql';
+import { gql } from "urql";
+import * as Urql from "urql";
```

Comment:
hmm, this file should be [ignored with `.prettierignore`](https://github.com/holic/web3-scaffold/blob/main/packages/app/.prettierignore) (since it's autogenerated), any idea why its still getting formatted?

Reply:
@holic I think my format on save changed this, I'll look into updating the VS Code project settings for that.

If this is auto-generated, would we want to consider dropping it from version control?

Reply:
I think checking in auto generated files is fine (like types) to ensure they're there and you don't need to run some build command before using the repo, we should just make sure they're not within the scope of autoformatting, etc. since they'll always be fighting each other.